data-raw/test-api.R

```diff
@@ -0,0 +1,53 @@
+library(httr)
+library(jsonlite)
+library(EDIutils)
+library(xml2)
+library(tidyr)
+
```

Comment:
When testing an api, you usually want to mock the endpoint instead of hitting the real server. A quick google turned these resources up:
https://discuss.ropensci.org/t/best-practices-for-testing-api-packages/460
https://ropensci.org/blog/2019/12/11/http-testing/
https://cran.r-project.org/web/packages/httptest/vignettes/httptest.html

Reply:
I created a new branch for my api tests.

Reply:
i got started on writing tests using the vcr package. I still need to complete the tests for the api functions